src/components/structures/UploadBar.tsx

```diff
@@ -16,16 +15,22 @@ limitations under the License.
 */
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import ContentMessages from '../../ContentMessages';
 import dis from "../../dispatcher/dispatcher";
 import filesize from "filesize";
 import { _t } from '../../languageHandler';
+import {Room} from "matrix-js-sdk/src/models/room";
```

Comment:
spacing consistency pls

Reply:
n        o

Reply:
at least put it at the top with the other absolute import :'(

Reply:
it was a funnier joke when the browser didn't collapse my spaces :(